internal/scan-manager/src/main/resources/scan-manager-config.yaml

```diff
@@ -0,0 +1,26 @@
+#Scan Manager Configuration
+
+databaseUrl : jdbc:mysql://localhost:3306/smdb?useSSL=false
+userName : root
+password : admin
```

Comment:
remove this value and keep placeholders.

Reply:
done